src/cache_client.rs

```diff
@@ -696,6 +699,161 @@ impl CacheClient {
         request.send(self).await
     }
 
+    /// Remove multiple elements from the sorted set.
+    ///
+    /// # Arguments
+    /// * `cache_name` - The name of the cache containing the sorted set.
+    /// * `sorted_set_name` - The name of the sorted set to remove elements from.
+    /// * `values` - The values to remove. Must be able to be converted to a `Vec<u8>`.
+    ///
+    /// # Examples
+    /// Assumes that a CacheClient named `cache_client` has been created and is available.
+    /// ```
+    /// # fn main() -> anyhow::Result<()> {
+    /// # use momento_test_util::create_doctest_cache_client;
+    /// # tokio_test::block_on(async {
+    /// use momento::cache::SortedSetRemoveElements;
+    /// # let (cache_client, cache_name) = create_doctest_cache_client();
+    /// let sorted_set_name = "sorted_set";
+    ///
+    /// let remove_elements_response = cache_client.sorted_set_remove_elements(
+    ///     cache_name,
+    ///     sorted_set_name,
+    ///     vec!["value1", "value2"]
+    /// ).await;
+    ///
+    /// match remove_elements_response {
+    ///     Ok(_) => println!("Elements removed from sorted set"),
+    ///     Err(e) => eprintln!("Error removing elements from sorted set: {}", e),
+    /// }
+    /// # Ok(())
+    /// # })
+    /// # }
+    /// ```
+    /// You can also use the [send_request](CacheClient::send_request) method to remove elements using a [SortedSetRemoveElementsRequest].
+    pub async fn sorted_set_remove_elements(
+        &self,
+        cache_name: impl Into<String>,
+        sorted_set_name: impl IntoBytes,
+        values: Vec<impl IntoBytes>,
```

Comment:
I will add an item to the backlog to abstract away "a list of intobytes" for the purposes of these collections. Functions like:
- dictionary get fields
- dictionary remove fields
- sorted set remove elements
- set add elements

etc all take a set of `IntoBytes`, so that's a common concern.